src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaverForBarnService.kt

```diff
@@ -53,10 +61,10 @@ class OpprettOppgaverForBarnService(private val oppgaveClient: OppgaveClient,
         val aktørId = familieIntegrasjonerClient.hentAktørId(oppgaveForBarn.personIdent)
         val finnOppgaveRequest = FinnOppgaveRequest(tema = Tema.ENF,
                                                     aktørId = aktørId,
-                                                    oppgavetype = Oppgavetype.InnhentDokumentasjon)
+                                                    fristFomDato = LocalDate.now().minusDays(1))
```

Comment:
Ettersom vi nå har 2 ulike oppgavetyper for dette, så satte jeg att fristferdigstilling skulle være fra dagens dato

Mulig man egentligen burde sette denne til å sette fra og til, for å også kunne matche siste tom-datoet ? 
```kotlin
val nå = oppgaveForBarn.aktivFra ?: LocalDate.now()
        val fristFomDato = nå.minusDays(1)
        val fristTomDato = nå.plusWeeks(2)
```

Reply:
Nytt førsøk:
Hvis jeg fikk ett barn for 12 måneder sen, så fyller det år snart
Men så får jeg et til barn innen det første fyller 12 måneder (vet ikke hvor ofte det skjer :baby:)
Då kommer vi opprette oppfølgningsoppgave for det nye barnen, som har frist om 6 og 12 måneder

Når det første barnet då fyller 1 år, så sjekker vi om det finnes en oppgave før vi oppretter
Hvis vi ikke setter fristTomDato i “finnOppgaver” så kommer vi finne oppgavene til det siste barnet, og då ikke opprette oppfølgningsoppgave for det barnet som fyller 1 år :stuck_out_tongue:

Reply:
Ved spørsmål om hva aktivFra brukes til, og om man i teorien kunne brukt den i stedet for tom-dato: "aktivFra settes idag alltid til dagens dato, hvis den settes frem i tiden blir den ikke vist i oppgavebenken".
Godkjenner :)